scripts/fuse/docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# run this container with any of the run scripts (in order to mount the source code folder, ensure privileged mode to be able to mount fuse, setup logging to nohup.out/stdout)
```

Comment:
Any reason why this is not in scripts/docker/?

Reply:
Imho because it serves the singular purpose of testing the fuse implementation interactively, therefore it should be where it is needed.
Of course they could be moved and long relative file paths to them introduced, but the benefit is not clear to me.

Reply:
It is only intended for developers, normal users only need the FUSE python dependency as described in README.md